src/styles/utils/generators/TooltipStyleUtils/index.ts

```diff
@@ -269,6 +276,21 @@ const createTooltipStyleUtils: StyleUtilGenerator<GetTooltipStylesStyleUtil> = (
             },
         };
     },
+
+    // Utility function to create and manage scale animations with React Native Reanimated
```

Comment:
Let's use JSDoc syntax, this way when you hover over functions you get intellisense 😄 
```suggestion
    /** Utility function to create and manage scale animations with React Native Reanimated */
```